api.md

```diff
@@ -1822,6 +1823,37 @@ ORDER BY day;
 
 ## Utilities/Statistics [](utilities)
 
+### timescaledb_information.data_node [](#timescaledb_information-datanode)
+
+Get information on data nodes. This function is specific to running
+TimescaleDB in a multi-node setup.
+
```

Comment:
Should be an h4 `#### Available Columns` here.  See `timescaledb_information.hypertable` below for example.

Reply:
Thanks for catching that